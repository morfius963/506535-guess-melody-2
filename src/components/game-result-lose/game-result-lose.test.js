import React from "react";
import renderer from "react-test-renderer";
import GameResultLose from "./game-result-lose.jsx";

jest.mock(`react-router-dom`, () => ({
  useHistory: () => ({
    push: jest.fn(),
  }),
}));

describe(`snapshot test`, () => {
  it(`Component correctly renders`, () => {
    const props = {
      result: `mistakes`,
      restartGame: jest.fn()
    };

    const tree = renderer
      .create(<GameResultLose {...props} />)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
