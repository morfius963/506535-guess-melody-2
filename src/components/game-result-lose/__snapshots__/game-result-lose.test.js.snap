// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot test Component correctly renders 1`] = `
<section
  className="result"
>
  <div
    className="result__logo"
  >
    <img
      alt="Угадай мелодию"
      height="83"
      src="img/melody-logo.png"
      width="186"
    />
  </div>
  <h2
    className="result__title"
  >
    Какая жалость!
  </h2>
  <p
    className="result__total result__total--fail"
  >
    Время вышло! Вы не успели отгадать все мелодии
  </p>
</section>
`;
